README.md

```diff
@@ -142,7 +142,7 @@ git-cliff [FLAGS] [OPTIONS] [--] [RANGE]
 -l, --latest        Processes the commits starting from the latest tag
     --current       Processes the commits that belong to the current tag
 -u, --unreleased    Processes the commits that do not belong to a tag
-    --topo-order    Sorts the tags topologically
+    --date-order    Sorts the tags chronologically.
```

Comment:
nitpick: can we remove the dot at the end?

Reply:
Nice catch! I think it's due to my copy-and-paste. :cry: 
Removed in 0775f05.